bufferserver/src/main/java/com/datatorrent/bufferserver/internal/LogicalNode.java

```diff
@@ -346,7 +346,9 @@ public void boot(EventLoop eventloop)
   @Override
   public String toString()
   {
-    return "LogicalNode{" + "upstream=" + upstream + ", group=" + group + ", partitions=" + partitions + ", iterator=" + iterator + '}';
+    return getClass().getName() + "@" + Integer.toHexString(hashCode()) +
```

Comment:
You will find that many toString implementations use the short name as it is sufficient to convey the  required information.